client/src/services/TransactionService.tsx

```diff
@@ -1,7 +1,7 @@
 /* eslint no-console: 0 */
 import { Transaction } from '../interfaces';
 
-const fetchRequest = (path: string, options?: any) => {
+const fetchRequest = (path: string, options?: {[key: string]: string | {}}) => {
```

Comment:
Does this need the `| {}`?  
(or could you solve this by not having options be optional, but specify a default parameter?